sql/branch.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE emails DROP CONSTRAINT emails_participant_fkey;
+
+ALTER TABLE emails ALTER COLUMN participant TYPE bigint USING participant::bigint;
```

Comment:
What's the effect of the `participant::bigint` cast? Does this properly convert the values in the `emails.participant` column?